svir/dialogs/viewer_dock.py

```diff
@@ -1162,23 +1162,33 @@ def clear_plot(self):
 
     def clear_imt_cbx(self):
         if self.imt_cbx is not None:
-            self.imt_cbx.blockSignals(True)
-            self.imt_cbx.clear()
-            self.imt_cbx.blockSignals(False)
+            try:
+                self.imt_cbx.blockSignals(True)
+                self.imt_cbx.clear()
+                self.imt_cbx.blockSignals(False)
+            except RuntimeError:
+                # continue if something like this occurs:
+                # "wrapped C/C++ object of type QComboBox has been deleted"
+                pass
```

Comment:
I would at least log the error, it is scary to hide all the RuntimeErrors

Reply:
Done